themis_lock.go

```diff
@@ -84,34 +101,36 @@ func (l *lock) parseField(r iohelper.ByteMultiReader) error {
 	return nil
 }
 
-func (l *lock) setColumn(col *hbase.ColumnCoordinate) {
-	l.column = col
-}
-
-func (l *lock) getColumn() *hbase.ColumnCoordinate {
-	return l.column
-}
-
-func (l *lock) setExpired(b bool) {
-	l.expired = b
-}
-
-func parseLockFromBytes(b []byte) (ThemisLock, error) {
+func parseLockFromBytes(b []byte) (Lock, error) {
 	buf := bytes.NewBuffer(b)
 	var isPrimary uint8
 	err := binary.Read(buf, binary.BigEndian, &isPrimary)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	var ret ThemisLock
+	var ret Lock
 	if isPrimary == 1 {
-		ret = newPrimaryLock()
+		l := newThemisPrimaryLock()
```

Comment:
you can use `ret = newThemisPrimaryLock()` directly and call parse out of `if-else`.